caster/lib/dfplus/merge/ccrmerger.py

```diff
@@ -299,7 +332,7 @@ def merge(self, time, name=None, enable=True, save=False):
             rule.set_context(context)
             active_apps.append(rule)
         '''negation context for appless version of base rule'''
-        contexts = [rule.get_context() for rule in self._app_rules.values() \
+        contexts = [app_rule.get_context() for app_rule in self._app_rules.values() \
                     if rule.get_context() is not None]# get all contexts
```

Comment:
Shouldn't this be?:
```suggestion
                    if app_rule.get_context() is not None]# get all contexts
```
or am I missing something?